src/services/lnd/index.ts

```diff
@@ -418,13 +454,20 @@ const lookupPaymentByPubkeyAndHash = async ({
 
     return paymentLookup
   } catch (err) {
-    return new UnknownLightningServiceError(err)
+    const errDetails = parseLndErrorDetails(err)
+    switch (errDetails) {
+      default:
+        return new UnknownLightningServiceError(err)
+    }
   }
 }
 
 const parseLndErrorDetails = (err) =>
```

Comment:
some documentation could be helpful here (maybe on the way `ln-service`/`lightning` manages error in a non conventional way.

Reply:
Added: https://github.com/GaloyMoney/galoy/pull/804